### PR TITLE
Script to validate directories changed in operator submission

### PR DESCRIPTION
**Pull request by MarcinGinszt**
Related PR: https://github.com/redhat-openshift-ecosystem/operator-pipelines-images/pull/5

Jira task: ISV-908

### CODE REVIEW

File: tasks/verify-changed-directories.yml [hunk 1]
```diff
@@ -6,14 +6,19 @@ metadata:
...
+      image: quay.io/redhat-isv/operator-pipelines-images:latest
       script: |
         #! /usr/bin/env bash
-        # curl https://api.github.com/repos/operator-framework/community-operators/compare/master...$(params.git_pr_branch)
-        # Operator name and version should correspond to the ancestor directory names. A new release directory should be created.
-        # There should be no additional changes.
-        echo "ok"
+        verify_changed_dirs \
+          --operator_name="$(params.operator_name)" \
+          --bundle-version="$(params.bundle_version)" \
```
Comment (Allda) [inline tasks/verify-changed-directories.yml:21]: the CLI has `--bundle_version` argument. Please replace `-` with `_`

Comment (amisstea) [reply tasks/verify-changed-directories.yml:21]: Would it be better to keep `-` and fix the other args instead? Maybe I'm just a big fan of dashes :smile:

Comment (MarcinGinszt) [reply tasks/verify-changed-directories.yml:22]: @amisstea  You are right, I changed the underscores to hyphens for consistency with other scripts parameters. Related PR: https://github.com/redhat-openshift-ecosystem/operator-pipelines-images/pull/8

File: README.md [hunk 2]
```diff
@@ -89,16 +88,20 @@ curl https://raw.githubusercontent.com/tektoncd/catalog/main/task/yaml-lint/0.1/
...
-  --param bundle_path=operators/kogito-operator/1.6.0 \
+  --param pr_head_label=MarcinGinszt:test-PR-ok \
+  --param bundle_path=operators/kogito-operator/1.6.1-ok \
   --param pyxis_url=https://catalog.redhat.com/api/containers \
   --param preflight_min_version=1.0.0 \
   --param ci_min_version=1.0.0 \
   --workspace name=repository,volumeClaimTemplateFile=templates/workspace-template.yml \
   --workspace name=results,volumeClaimTemplateFile=templates/workspace-template.yml \
+  --workspace name=ssh-dir,secret=my-ssh-credentials \
+  --workspace name=registry-credentials,secret=redhat-isv-redhat-isv-robot-pull-secret \
```
[the remark this answers is not in the document]
Comment (MarcinGinszt) [reply README.md]: done

File: pipelines/operator-hosted-pipeline.yml [hunk 3]
```diff
@@ -220,6 +236,8 @@ spec:
         - name: source
           workspace: repository
           subPath: src
+        - name: credentials
+          workspace: registry-credentials
```
[the remark this answers is not in the document]
Comment (MarcinGinszt) [reply pipelines/operator-hosted-pipeline.yml:244]: done

File: pipelines/operator-hosted-pipeline.yml [hunk 4]
```diff
@@ -18,9 +20,15 @@ spec:
       default: quay.io
     - name: image_stream
       default: $(context.pipelineRun.namespace)
+    - name: test_mode
```
Comment (Allda) [inline pipelines/operator-hosted-pipeline.yml:23]: The test mode should be only in the ci pipeline to allow quicker bundle development, hosted should always run all tasks.

Comment (MarcinGinszt) [reply pipelines/operator-hosted-pipeline.yml]: I set the "skip" to default "false" in the `diggest-pinning` task and removed this argument from the hosted pipeline.

**Review by Allda — APPROVED** [review]

**Pull request merged**